ozaria/site/components/teacher-dashboard/common/_moon-button.scss

```
@@ -0,0 +1,26 @@
+.moon-btn {
+  background-color: $moon;
+  border-radius: 4px;
+  border-width: 0;
+  text-shadow: unset;
+  @include font-p-3-small-button-text-black;
+  background-image: unset;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)